File: sivemor-app/src/main/java/mx/edu/utez/sivemorapp/modules/reportes/dto/ReporteOpcionesDTO.java
package mx.edu.utez.sivemorapp.modules.reportes.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ReporteOpcionesDTO {
    private List<OpcionDto> clientes;
    private List<OpcionDto> regiones;
    private List<OpcionDto> notas;
    private List<String> tiposVerificacion;
    private List<String> dictamenes;

    @Data
    @NoArgsConstructor
    @AllArgsConstructor
    public static class OpcionDto {
        private Long id;
        private String nombre;
    }
}

File: SivemorMobile/app/src/main/java/com/sivemor/mobile/network/dto/ReporteOpcionesDTO.kt
package com.sivemor.mobile.network.dto

data class ReporteOpcionesDTO(
    val clientes: List<OpcionDto> = emptyList(),
    val regiones: List<OpcionDto> = emptyList(),
    val notas: List<OpcionDto> = emptyList(),
    val tiposVerificacion: List<String> = emptyList(),
    val dictamenes: List<String> = emptyList()
)

data class OpcionDto(
    val id: Long,
    val nombre: String
)